MediaManager/Plugin.MediaManager.iOS/MediaManagerImplementation.cs

```diff
@@ -104,6 +106,27 @@ protected virtual void OnBuffering(EventArgs e)
             if (Buffering != null)
                 Buffering(this, e);
         }
+
+        public async Task Play(IMediaFile mediaFile)
+        {
+            switch (mediaFile.Type)
+            {
+                case MediaFileType.AudioUrl:
+                    await Play(mediaFile.Url);
+                    break;
+                case MediaFileType.VideoUrl:
+                    break;

```

Comment:
Why not throw an NotImplemented exception for now, because no implementation exists?